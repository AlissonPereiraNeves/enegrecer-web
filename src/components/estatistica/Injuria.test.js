import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Injuria from './Injuria';

describe('Injuria', () => {
  it('deve construir a pagina corretamente ', () => {
    const injuria = shallow(<Injuria />);
    const json = toJson(injuria);
    expect(json).toMatchSnapshot();
  });
});
