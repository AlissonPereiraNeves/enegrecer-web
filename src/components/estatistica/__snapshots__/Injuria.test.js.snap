// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Injuria deve construir a pagina corretamente  1`] = `
<TipoDeEstatistica
  categoria="InjÃºria Racial"
  id="amarelo"
  numero="0"
/>
`;
